.travis.yml

```diff
@@ -23,21 +23,27 @@ matrix:
       language: generic
       python: 2.7
 
+addons:
+  homebrew:
+    packages:
+    - hdf5
+    update: true
+  apt:
+    packages:
+    - libhdf5-dev
```

Comment:
\o/

Reply:
Do you have experience with this? I'm surprised it makes things slower.

Reply:
No, just seems a lot better than having to "script" this…

Reply:
Turning off the 'update' makes it faster, based on a sample size of one.

Reply:
What if you keep the Homebrew cache ?

Reply:
There are a bunch of them, and the process for handling it seems error prone:
https://stackoverflow.com/a/53331571

I'd prefer to use what travis gives automatically, since they probably know best what works/ doesn't work, and I don't want to spend hours debugging it.